providers/process/abstractProcessor.js

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const BaseHandler = require('../../lib/baseHandler')
+const EntitySpec = require('../../lib/entitySpec')
+const fs = require('fs')
+const path = require('path')
+const shajs = require('sha.js')
+const { intersection, pick, set } = require('lodash')
+const { promisify } = require('util')
+const nodeDir = promisify(require('node-dir').files)
+const { trimAllParents } = require('../../lib/utils')
+
+class AbstractProcessor extends BaseHandler {
+  get schemaVersion() {
+    return '0.0.0'
```

Comment:
suggest we bump this to start so we get 1 version up for the `attachments` schema change
```suggestion
    return '0.1.0'
```